TmdbEasy.Tests/RequestHandlerTests.cs

```diff
@@ -1,55 +1,33 @@
 ﻿using NSubstitute;
 using NUnit.Framework;
 using System.Threading.Tasks;
+using TmdbEasy.Configurations;
 using TmdbEasy.Interfaces;
 
 namespace TmdbEasy.Tests
 {
     [TestFixture]
     public class RequestHandlerTests
     {
-        private readonly ITmdbEasyClient _subClient;
-
-        public RequestHandlerTests()
-        {
-            _subClient = Substitute.For<ITmdbEasyClient>();
-        }
-
-        [Test]
-        public void CreateRequest_CreatesValidRequest_SetsBaseUrl()
-        {
-            string testBaseUrl = "https://baseurl";
-
-            _subClient.GetBaseUrl().Returns(testBaseUrl);
-
-            var handlerUnderTest = new RequestHandler(_subClient);
-
-            Request request = handlerUnderTest.CreateRequest();
-            
-            string result = request.GetUri();
-
-            Assert.AreEqual(testBaseUrl, result);
-        }
```

Comment:
I think i made a mistake by deleting this test case.

Reply:
Yeap lets include that again

Reply:
I have to confess i wrote a test case for the `CreateRequest` method but it only checks that it won't return null.

Reply:
Thats fine